extensions/spring-di/deployment/pom.xml

```diff
@@ -38,12 +38,22 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-arc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-undertow-runtime</artifactId>
```

Comment:
@BrentDouglas The session context is registered in the undertow extension but you need to depend on the deployment module to make it work -> `io.quarkus:quarkus-undertow`.

Reply:
Ok, Updated.